impl/src/main/java/org/jboss/weld/logging/BeanLogger.java

```diff
@@ -522,4 +522,8 @@
     @Message(id = 1570, value = "Invalid BeanConfigurator setup - no callback was specified for {0}", format = Format.MESSAGE_FORMAT)
     IllegalStateException noCallbackSpecifiedForCustomBean(Object bean);
 
+    @LogMessage(level = Level.WARN)
```

Comment:
I was thinking if `INFO` wouldn't be more appropriate log level? Just because existing applications (which had, say, `null` package) will now get warning without changing anything. Also sealing a package/signing a class is not a bad practice and you shouldn't get warning for that.